packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -87,6 +87,11 @@ export default class UpdateConfigure extends EasCommand {
       Log.withTick(`Configured ${chalk.bold('Expo.plist')} for EAS Update`);
     }
 
+    Log.addNewLineIfNone();
+    Log.warn(
+      `If you have previously deployed this app to users, you will need to rebuild and submit a new version before you can update your app with EAS Update.`
```

Comment:
```suggestion
      `All builds of your app going forward will be eligible to receive updates published with EAS Update.`
```

My suggestion might be a bit technical, I'm not sure. Would be good to get feedback from other reviewers. I think the thing I'd wan't this warning to convey is that only new builds that have this configuration will get updates. Maybe we just say that? idk.

Reply:
I think your suggestion states a fact that the user likely understands at this point in time, whereas the original warning's intention is to make the user aware that they might have to take an extra action to get the outcome they desire

Reply:
`All builds of your app` might worry users that somehow all builds will be getting all updates, to avoid that maybe we could change `All` -> `Only`. Of course, it might be desired behavior to introduce a bit of uncertainty to make users check docs.